src/components/execution-environment/repository-form.tsx

```diff
@@ -349,6 +357,22 @@ export class RepositoryForm extends React.Component<IProps, IState> {
     );
   }
 
+  private validateName(name) {
+    const regex = new RegExp('^[0-9A-Za-z._-]*/?[0-9A-Za-z._-]*$');
```

Comment:
```suggestion
    const regex = /^([0-9A-Za-z._-]+\/)?[0-9A-Za-z._-]+$/;
```

(javascript has regex literals, so no need to use a string; and wrapping the part up to and including the slash lets us have the whole bit optional and then we can use `+` to require at least one character before/after the slash (prevents `/foo` or `foo/` or `/` from being valid))